GSA/digitalgov.gov
# Add ga4 to dap common questions

## Summary

Added the DAP GA4 information to the common questions section of the guide. We will need to create another PR to add images and alt text. 

### Preview

[Link to Preview]()



### Solution

### How To Test

1. Check that the summary box has rendered the headings correctly
2. Check links and formatting
3. Check spelling

---

### Dev Checklist
